### PR TITLE
Adds New SCC ERT Hardsuits for Admin & Event Use

### PR DESCRIPTION
**Pull request by io-util**
Does what it says in the title. This PR adds new SCC themed ERT Commander, Engineer, Medic and Security hardsuits for use by Admins within events. The hardsuits not naturally obtainable and do not spawn in the Centcom ERT Armory, the Armory still gets the NT ERT Hardsuits.

Request to be added comes from Read, sprites have been created by Kyres. Balance wise, they're supposed to be stronger (and it shouldn't matter, since they are adminspawn only). 

The Admin Outfits use the Security Officer uniforms because of the issue with the suit debug sprite showing on non-horizon undersuits.

Screenshot of the Hardsuits:

scc_ert

### CODE REVIEW

**Comment by io-util** [comment]
!review

**Comment by Eyeveri** [comment]
These are needed for the event on Saturday, August 19. 

Caelphon has approved this to be used for non-canon reasons. 
Cael

File: html/changelogs/scc-erthardsuits-io-util.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: io-util
+
+delete-after: True
+
+changes:
+  - rscadd: "Added new SCC ERT Hardsuits for Event and Admin use"
+  - imageadd: "Added new sprites for the SCC ERT Hardsuits (thanks kyres)"
```
Comment (SleepyGemmy) [inline html/changelogs/scc-erthardsuits-io-util.yml:7]:
```suggestion
  - rscadd: "Added new SCC ERT Hardsuits for Event and Admin use."
  - imageadd: "Added new sprites, made by Kyres, for the SCC ERT Hardsuits."
```

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 2]
```diff
@@ -118,6 +118,60 @@
...
+	name = "SCC ERT-M hardsuit control module"
+	desc = "A suit worn by the medical specialist of an Emergency Response Team sent to crisis situations by the Stellar Corporate Conglomerate. This one features SCC colouring."
+	suit_type = "SCC ERT medic"
+	icon = 'icons/clothing/rig/scc_ert/scc_medic.dmi'
+	icon_state = "scc_rig"
+
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL | MODULE_MEDICAL
+
+/obj/item/rig/ert/scc/security
+	name = "SCC ERT-S hardsuit control module"
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:167]:
```suggestion
	name = "\improper SCC ERT-S hardsuit control module"
```

File: html/changelogs/scc-erthardsuits-io-util.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+author: io-util
+
+delete-after: True
+
+changes:
+  - rscadd: "Added new SCC ERT Hardsuits for event and admin use."
+  - imageadd: "Added new sprites, made by Kyres, for the SCC ERT Hardsuits."
```
Comment (SleepyGemmy) [inline html/changelogs/scc-erthardsuits-io-util.yml:7]:
```suggestion
  - imageadd: "Added new sprites, made by Kyres, for the SCC ERT hardsuits."
```

**Review by SleepyGemmy — COMMENTED** [review]
the icon files should preferably be the same or similar to their item paths, i.e. `icon = 'icons/obj/item/rig/ert/scc.dmi'`

you can put all the rigs inside this singular suggested file, but if you wish you can also do it like
`icon = 'icons/obj/item/rig/ert/scc/commander.dmi'`, `icon = 'icons/obj/item/rig/ert/scc/medical.dmi'`, etc.

**Comment by io-util** [comment]
> the icon files should preferably be the same or similar to their item paths, i.e. `icon = 'icons/obj/item/rig/ert/scc.dmi'`. if you wish you can also do it like `icon = 'icons/obj/item/rig/ert/scc/commander.dmi'`, `icon = 'icons/obj/item/rig/ert/scc/medical.dmi'`, etc.

Done, I can throw all the icon states into the same `.dmi` after renaming them if you'd prefer it, but this works.

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 4]
```diff
@@ -118,8 +118,62 @@
 		)
 	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL
 
+/obj/item/rig/ert/scc
+	name = "\improper SCC ERT-C hardsuit control module"
+	desc = "A suit worn by the commander of an Emergency Response Team sent to crisis situations by the Stellar Corporate Conglomerate. This one features SCC colouring."
+	suit_type = "SCC ERT commander"
+	icon = 'icons/clothing/rig/scc_ert/commander.dmi'
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:125]:
```suggestion
	icon = 'icons/obj/item/rig/ert/scc/commander.dmi'
```

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 5]
```diff
@@ -118,8 +118,62 @@
...
+		/obj/item/rig_module/maneuvering_jets,
+		/obj/item/rig_module/actuators/combat,
+		/obj/item/rig_module/storage
+		)
+
+/obj/item/rig/ert/scc/engineer
+	name = "\improper SCC ERT-E hardsuit control module"
+	desc = "A suit worn by the engineer of an Emergency Response Team sent to crisis situations by the Stellar Corporate Conglomerate. This one features SCC colouring."
+	suit_type = "SCC ERT engineer"
+	icon = 'icons/clothing/rig/scc_ert/engineer.dmi'
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:152]:
```suggestion
	icon = 'icons/obj/item/rig/ert/scc/engineer.dmi'
```

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 6]
```diff
@@ -118,8 +118,62 @@
...
+	icon_state = "scc_rig"
+
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL | MODULE_UTILITY
+
+/obj/item/rig/ert/scc/medical
+	name = "\improper SCC ERT-M hardsuit control module"
+	desc = "A suit worn by the medical specialist of an Emergency Response Team sent to crisis situations by the Stellar Corporate Conglomerate. This one features SCC colouring."
+	suit_type = "SCC ERT medic"
+	icon = 'icons/clothing/rig/scc_ert/medic.dmi'
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:161]:
```suggestion
	icon = 'icons/obj/item/rig/ert/scc/medic.dmi'
```

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 7]
```diff
@@ -118,8 +118,62 @@
...
+	icon_state = "scc_rig"
+
+	allowed_module_types = MODULE_GENERAL | MODULE_LIGHT_COMBAT | MODULE_HEAVY_COMBAT | MODULE_SPECIAL | MODULE_MEDICAL
+
+/obj/item/rig/ert/scc/security
+	name = "\improper SCC ERT-S hardsuit control module"
+	desc = "A suit worn by the security specialist of an Emergency Response Team sent to crisis situations by the Stellar Corporate Conglomerate. This one features SCC colouring."
+	suit_type = "SCC ERT security"
+	icon = 'icons/clothing/rig/scc_ert/security.dmi'
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:170]:
```suggestion
	icon = 'icons/obj/item/rig/ert/scc/security.dmi'
```

File: icons/clothing/rig/scc_ert/commander.dmi
Comment (SleepyGemmy) [inline icons/clothing/rig/scc_ert/commander.dmi:1]: `icons/obj/item/rig/ert/scc/commander.dmi`

File: icons/clothing/rig/scc_ert/engineer.dmi
Comment (SleepyGemmy) [inline icons/clothing/rig/scc_ert/engineer.dmi:1]: `icons/obj/item/rig/ert/scc/engineer.dmi`

File: icons/clothing/rig/scc_ert/medic.dmi
Comment (SleepyGemmy) [inline icons/clothing/rig/scc_ert/medic.dmi:1]: `icons/obj/item/rig/ert/scc/medic.dmi`

File: icons/clothing/rig/scc_ert/security.dmi
Comment (SleepyGemmy) [inline icons/clothing/rig/scc_ert/security.dmi:1]: `icons/obj/item/rig/ert/scc/security.dmi`

**Comment by SleepyGemmy** [comment]
because its a similar path to the actual item's, which makes the DMI easier to find.

**Comment by io-util** [comment]
> because its a similar path to the actual item's, which makes the DMI easier to find.

I get that, but I think it's rather counter-productive to move only those sprites there, if all of the other hardsuit sprites stay where they currently are, unless those are planned to be moved too

**Comment by SleepyGemmy** [comment]
new additions should preferably use more updated methods, but it works either way.

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**